### PR TITLE
Scope tokens

### PR DESCRIPTION
**Pull request by sgryt**
This is a companion PR to https://github.com/GreanTech/identityprovider/pull/445, soliciting feedback about placement, searchability, etc.

### CODE REVIEW

File: src/pages/verify/e-ids/danish-mitid.mdx [hunk 1]
```diff
@@ -39,6 +39,25 @@ You create personal MitID test users at [https://pp.mitid.dk/test-tool/frontend]
 
 ![MitID test tool](./images/mitid-testtool.png)
 
+## Available data / scopes
```
Comment (mickhansen) [inline src/pages/verify/e-ids/danish-mitid.mdx:42]: I think the headline should be something like `Collecting user addresses` instead. (For NemID as well)

**Comment by mickhansen** [comment]
We should add a general `dynamic scopes/data` page or expand the URL builder so we can link to that page from the applications management UI

File: src/pages/verify/e-ids/danish-mitid.mdx (hunk 1 above)
Comment (sgryt) [reply src/pages/verify/e-ids/danish-mitid.mdx]: Updated

**Comment by sgryt** [comment]
> We should add a general `dynamic scopes/data` page or expand the URL builder so we can link to that page from the applications management UI

Expanding the URL builder sounds like a good plan - but I'm not sure if it becomes too busy with (more) sub-options per e-ID ?

**Comment by mickhansen** [comment]
> > We should add a general `dynamic scopes/data` page or expand the URL builder so we can link to that page from the applications management UI
> 
> Expanding the URL builder sounds like a good plan - but I'm not sure if it becomes too busy with (more) sub-options per e-ID ?

Yeah maybe, it is already quite busy.

**Comment by sgryt** [comment]
> > > We should add a general `dynamic scopes/data` page or expand the URL builder so we can link to that page from the applications management UI
> > 
> > 
> > Expanding the URL builder sounds like a good plan - but I'm not sure if it becomes too busy with (more) sub-options per e-ID ?
> 
> Yeah maybe, it is already quite busy. But seems like a good place to have it.

Certainly.

**Comment by mickhansen** [comment]
Deploy preview somehow died, but LGTM

**Comment by mickhansen** [comment]
@sgryt Ready for merge?

**Comment by sgryt** [comment]
@mickhansen Ready for review.

**Comment by sgryt** [comment]
@mickhansen Also ready for review.

**Pull request merged**